priiloader/source/Input.cpp

```diff
@@ -22,14 +22,70 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 #include <gccore.h>
 #include <wiiuse/wpad.h>
+#include <wiikeyboard/keyboard.h>
 #include <sys/time.h>
+#include <sys/unistd.h>
 
 #include "Input.h"
 
 s8 _input_init = 0;
 u32 _STM_input_pressed = 0;
+u32 _kbd_pressed = 0;
 struct timeval _last_press;
 struct timeval _time_init;
+static lwp_t kbd_handle = LWP_THREAD_NULL;
+static volatile bool kbd_should_quit = false;
+
+void HandleKBDEvent(char sym) {
+	switch (sym) {
+		case 0x84:
+			_kbd_pressed |= INPUT_BUTTON_UP;
+			break;
+		case 0x85:
+			_kbd_pressed |= INPUT_BUTTON_DOWN;
+			break;
+		case 0x86:
+			_kbd_pressed |= INPUT_BUTTON_LEFT;
+			break;
+		case 0x87:
+			_kbd_pressed |= INPUT_BUTTON_RIGHT;
+			break;
+		case 'a':
```

Comment:
can Q & q be added here?
this is to 'support' qwery & qwertz (french and german layout keyboards) without causing much confusion to a user.
if there is a better way to detect the layout of the keyboard it would be better, but afaik there is not.

Reply:
why do you want "q" as a replacement for A? how is that going to help quertz and qwerty users? doesn't this library use qwerty by default, which would make all of a, b, x, y work out of the box?

Reply:
my mistake. i meant add Q to support AZERTY ( french and belgium keyboards ).

Reply:
Could be misleading though, if someone with a qwerty keyboard tries to use "q" for "quit" to exit / cancel something ... I'd leave it as-is and maybe add a note somewhere that priiloader assumes qwerty / qwertz.

Reply:
Sure thing, so 'Q' and 'q' should be added to INPUT_BUTTON_A? Anything else that would be needed to support those layouts?

Reply:
I would rather recommend something like when the user presses an unassigned button like q, pop up a hint somewhere that tells the user what keys to use. As I said above, making "q" (a key that is very often used to cancel / exit / abort something) be the default key to accept an action can be very confusing and cause conflicts, in my opinion.

Reply:
Leseratte makes a fair point there... 
maybe we could add AZERTY as a setting? default being qwerty/qwertz and azerty if its enabled in settings? 
its just that i can see people getting confused in both cases (no azerty support or q added as a )